src/pages/Result.vue

```diff
@@ -1,23 +1,32 @@
 <template>
-  <v-container id="container-results">
-    <v-row v-for="resource in filteredList " :key="resource.name">
-      <v-col>
-          <v-card class="mx-auto" max-width="344">
-            <v-card-text>
-              <p class="display-1 text--primary"> {{ resource.name }}</p>
-              <div class="text--primary"> 
-                <span v-html="resource.content"></span>
-              </div>
-              <div> 
-                      <v-chip v-for="tag in resource.tags" :key="tag" class="ma-2">
-                        {{ tag }}
-                      </v-chip>
-              </div>
-            </v-card-text>
-        </v-card>
-      </v-col>
-    </v-row>
-  </v-container>
+  <div>
+    <v-container>
+      <v-row>
+        <v-col align="left">
+          <v-btn id="btn-restart-assessment" @click.native="restart">Start Again</v-btn>
+        </v-col>
+      </v-row>
+    </v-container>
+    <v-container id="container-results">
+      <v-row v-for="resource in filteredList " :key="resource.name">
```

Comment:
again, I'm not _sure_ you need to use the grid system here but I'm not _that_ familiar with Vuetify layouts, I'll admit